### ISSUE TITLE
ICS calendar of events

### ISSUE THREAD

**Issue by takluyver** [southampton-python/southampton-python.github.io#5]
It would be nice if people could subscribe to a calendar of SPUG events.

What tools are there to generate one? How can we make it easy to update it along with the visible website?

One option would be to use a hosted service like meetup.com, but that requires a regular payment, which the group isn't really set up to handle.

**Comment by takluyver** [comment]
@rpep suggested using Google Calendar to produce an ICS

**Comment by popey** [comment]
Yeah, something that generates an online ICS would be great to allow others to subscribe in their calendar of choice. Thanks!
